fearless/Modules/AccountCreate/AccountCreateViewController.swift

```diff
@@ -156,15 +193,21 @@ final class AccountCreateViewController: UIViewController {
         presenter.proceed()
     }
 
-    @IBAction private func actionTextFieldEditingChanged() {
-        if derivationPathModel?.inputHandler.value != derivationPathField.text {
-            derivationPathField.text = derivationPathModel?.inputHandler.value
+    @IBAction private func substrateTextFieldEditingChanged() {
+        if substrateDerivationPathModel?.inputHandler.value != substrateDerivationPathField.text {
```

Comment:
is this validation really required ?

Reply:
There is legacy code from former team. To be honest - have no idea why we even should change field.text on this event